Using pretrained: True
number of unique_entities -> 1861
number of unique_entities -> 1861
number of unique_entities -> 1861
Training size 2 Val size 2 Test size 2
Traceback (most recent call last):
  File "main.py", line 135, in <module>
    Corpus_, entity_embeddings, relation_embeddings = load_data(args)
  File "main.py", line 113, in load_data
    None)
  File "/scratch/bz1030/relationPrediction/preprocess.py", line 31, in init_embeddings
    with open(entity_file) as f:
FileNotFoundError: [Errno 2] No such file or directory: './data/drugbank_1861/entity2vec.txt'
/opt/slurm/data/slurmd/job5773880/slurm_script: line 18: --load_gat: command not found
